<?xml version="1.0"?>
<launch>
  <node pkg="pcl_tutorial" type="plane_segmentation_node" name="plane_segmentation" output="screen">
    <remap from="point_cloud_in" to="/camera/depth_registered/points"/>
    <remap from="point_cloud_out" to="/plane_segmentation/points_cloud_out"/>
    <param name="dist_thresh" type="double" value="0.01" />
    <param name="max_iterations" type="double" value="50" />
  </node>
</launch>
